### ISSUE TITLE
Issue Reporter doesn't have a placeholder in the Extension drop down

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#193491]
![Image](https://github.com/microsoft/vscode/assets/2644648/dfa5caf6-cb4e-489c-83a4-6dd5afdc4e28)

This is needed for accessibility.
